src/theme/index.tsx

```diff
@@ -201,6 +201,7 @@ export function betaColors(darkMode: boolean): BetaColors {
     bg6: darkMode ? '#1c1c1c' : '#FFFFFF',
     bg7: darkMode ? '#2C2D33' : '#FFFFFF',
     bg8: darkMode ? '#212427' : '#FFFFFF',
+    bg9: darkMode ? '#454545' : '#CED0D9',
```

Comment:
where are we using this bg9 new color ?

Reply:
are used in selected card on "my portfolio"
https://github.com/prodesert22/components/blob/27b71e60f86de7b2bca67f1e3ead6411e26685fc/src/components/MyPortfolio/styleds.tsx#L29